### PR TITLE
VACMS-18954: CLP node view should match node edit

### PR DESCRIPTION
**Pull request by davidmpickett**
## Description

Relates to #18954

Campaign landing pages have 11 different "panels" (field groups). 4 of them are **required** and the other 7 are optional. This is the order they appear on the Node Edit view
- **Hero banner** 
- **Why this matters** 
- **What you can do** 
- Video 
- Spotlight
- Stories 
- Downloadable resources 
- Events 
- FAQs 
- Connect with us 
- **VA Benefits** 

## Testing done
Manual on Tugboat

## Screenshots

<details><summary>Everything optional enabled</summary>

**Prod example**

screencapture-prod-cms-va-gov-initiatives-veterans-experience-action-centers-2024-09-06-13_31_45

**Tugboat example**

screencapture-pr19138-3xaxclzrfwacwdouylpmulrrhdlrcvqp-ci-cms-va-gov-initiatives-veterans-experience-action-centers-2024-09-06-13_31_02

</details>

<details><summary>Nothing optional enabled</summary> 

**Prod example**

screencapture-prod-cms-va-gov-initiatives-veteran-childcare-assistance-program-2024-09-06-13_35_26

**Tugboat example**

screencapture-pr19138-3xaxclzrfwacwdouylpmulrrhdlrcvqp-ci-cms-va-gov-initiatives-veteran-childcare-assistance-program-2024-09-06-13_34_37

</details>

## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

As a Content Admin (e.g. david.pickett@va.gov)
1. Log in to [tugboat](https://pr19138-3xaxclzrfwacwdouylpmulrrhdlrcvqp.ci.cms.va.gov/)
2. **Create a [new Campaign Landing Page](https://pr19138-3xaxclzrfwacwdouylpmulrrhdlrcvqp.ci.cms.va.gov/node/add/campaign_landing_page) (minimal)**
   - [ ] Only fill in the required fields in the Hero banner, Why this matters,  What you can do, and VA benefits sections
   - [ ] Save the node
   - [ ] Validate that the Node View matches the fields populated on Node Edit
3. **Create a [new CLP](https://pr19138-3xaxclzrfwacwdouylpmulrrhdlrcvqp.ci.cms.va.gov/node/add/campaign_landing_page)(maximal)**
   - [ ] Fill in every field in every section
   - [ ] Save the node
   - [ ] Validate that the Node View matches the fields populated on Node Edit
4. **Enable optional sections** 
   - [ ] Edit the Minimal CLP you created in step 2
   - [ ] Enable one optional section e.g. Video
   - [ ] Fill out all required fields
   - [ ] Save the node
   - [ ] Confirm that Node View updates with only the new enabled section
   - [ ] Repeat this process for the 6 other optional sections: Spotlight, Stories, Downloadable resources, Events, FAQs, Connect with us
5. **Disable optional sections** 
   - [ ] Edit the Maximal CLP you created in step 3
   - [ ] Disable one optional section e.g. Video
   - [ ] Save the node
   - [ ] Expected behavior for each section  ([per amended AC](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18954#issuecomment-2338921217))
      - [ ] Video ⚠️ - still shows fields 
      - [ ] Spotlight ⚠️ - still shows fields 
      - [ ] Stories ⛔ - will throw an error unless you remove the Featured store before disabling  
      - [ ] Downloadable resources ⚠️ - still shows fields
      - [ ] Events ⚠️ - still shows fields
      - [ ] FAQs ⚠️ - still shows fields
      - [ ] Connect with us ✅ - Disappears!

## Dave's QA results matrix

Scenario | Current prod behavior | Tugboat behavior
-- | -- | --
Create a new CLP with no optional panels | Hero banner ✅ <br> Why this matters ✅ <br> What can you do ⚠️ <br>  VA Benefits ⚠️  | Hero banner ✅ <br> Why this matters to you ✅ <br> What you can do ✅ <br>  VA Benefits ✅ 
Create a new CLP with all optional panels |  Hero banner ✅ <br> Why this matters ✅ <br> What can you do ⚠️ <br>  Video ⛔ <br> Spotlight ⛔ <br> Stories ⛔ <br> Downloadable resources ⛔ <br> Events ⛔ <br> FAQs ⚠️  <br> Connect with us ⚠️ <br> VA Benefits ⚠️ | Hero banner ✅ <br> Why this matters ✅ <br> What you can do ✅  <br>  Video ✅  <br> Spotlight ✅  <br> Stories ✅  <br> Downloadable resources ✅  <br> Events ✅  <br> FAQs ✅   <br> Connect with us ✅ <br> VA Benefits ✅ 
Enable an optional panel on an existing CLP |  Video ⛔ <br> Spotlight ⛔ <br> Stories ⛔ <br> Downloadable resources ⛔ <br> Events ⛔ <br> FAQs ⚠️  <br> Connect with us ⚠️ | Video ✅ <br> Spotlight ⚠️  <br> Stories ✅<br> Downloadable resources ✅ <br> Events ✅ <br> FAQs ✅  <br> Connect with us ✅
Disable an optional panel on an existing CLP | Video ✅ <br> Spotlight ✅  <br> Stories ⛔ <br> Downloadable resources ✅ <br> Events ✅ <br> FAQs ⚠️ <br> Connect with us ⚠️ | Video ⚠️ <br> Spotlight ⚠️  <br> Stories ⛔  <br> Downloadable resources ⚠️ <br> Events ⚠️<br> FAQs ⚠️ <br> Connect with us ✅

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [x] Automated tests have passed.
- [x] Code Quality Tests have passed.
- [x] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### CODE REVIEW

File: .github/workflows/actions-metrics.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     timeout-minutes: 10
     steps:
       - name: Send GitHub Actions metrics to DataDog
-        uses: int128/datadog-actions-metrics@9eaafc87f92290ca061b8c6cde62821196444136 # v1.94.0
+        uses: int128/datadog-actions-metrics@5dbc8b81eb6d1b3cbd24099ff83373df472e0e36 # v1.95.0
```
Comment (dsasser) [inline .github/workflows/actions-metrics.yml:14]: Is this an intentional change? If so, can you help me understand why this change is necessary?

**Comment by davidmpickett** [comment]
Closing in favor of a fresh branch & PR
